Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:lts-alpine
+FROM node:14-alpine
```

Comment:
i would rather do this in separate PR, as it is an important change and not directly related to CI